pkg/service/test/test.go

```diff
@@ -260,6 +266,9 @@ func (t *tester) Test(path string, testReportPath string, appCmd string, options
 			break
 		}
 	}
+
+	pp.Printf("\n <=========================================> \n  COMPLETE TESTRUN SUMMARY. \n\tTotal tests: %s\n"+"\tTotal test passed: %s\n"+"\tTotal test failed: %s\n <=========================================> \n\n", totalTests, totalTestPassed, totalTestFailed)
+
 	t.logger.Info("test run completed", zap.Bool("passed overall", result))
 	// log the overall code coverage for the test run of go binaries
```

Comment:
please display total number of test suites run and their names  as well

Reply:
![Uploading image.png…]()